seminars/talk.py

```diff
@@ -53,6 +53,7 @@ def __init__(
             self.token = "%016x" % random.randrange(16 ** 16)
             self.display = seminar.display
             self.online = getattr(seminar, "online", bool(seminar.live_link))
+            self.tz = seminar.tz
```

Comment:
Properties don't work this way (Sage `lazy_attributes` do).  If you make `tz` a property, you need a setter in order to assign to it.  Maybe set `self.timezone = seminar.timezone` instead?

Reply:
Sorry, I just forgot to change that line, fixed.